mypy.ini

```diff
@@ -0,0 +1,7 @@
+[mypy]
+check_untyped_defs = True
+disallow_any_generics = True
+
+incremental = True
+scripts_are_modules = True
+#show_traceback = True  # Used in Zulip core; use here?
```

Comment:
We should, I think; it's mostly an option that helps with debugging if mypy crashes.